alert/alert.go

```diff
@@ -51,10 +51,11 @@ type Alert struct {
 	Datasource   string
 	DashboardUID string
 	PanelID      string
+	PanelName    string
 }
 
 // New creates a new alert.
-func New(name string, options ...Option) *Alert {
+func New(name, panelName string, options ...Option) *Alert {
```

Comment:
I think you need to add option here (like: `WithPanelName`) instead of adding it as arg

Reply:
It's not optional parameter, without panel name we can't match alert with panel